cpp/mysql_list_tables.cc

```diff
@@ -75,6 +84,8 @@ int Main(int argc, char *argv[]) {
 
     ProcessTablesOrViews(db_connection.get(), /* process_tables = */true);
     ProcessTablesOrViews(db_connection.get(), /* process_tables = */false);
+    ProcessTriggers(db_connection.get(), "vufind");
```

Comment:
database names should not be hardcoded

Reply:
I think it is dangerous to include an uncontrolled list of databases here because we may want to install different software on different servers and some of that software may have its own database tables with triggers etc.  Also, we currently only do table versioning and comparisons on ub_tools.

Reply:
Not hardcoded any longer.